frontend/components/EventPage/TimeLeft.tsx

```diff
@@ -0,0 +1,21 @@
+
+import { ReactElement } from 'react'
+import styled from 'styled-components'
+import TimeAgo  from 'react-timeago';
+
+import { LIGHT_GRAY } from '../../constants'
+
+const TimeLeft = ({
+    start,
+    className,
+  }: {
+    start: Date
+    className?: string
+  }): ReactElement => (
+    <TimeAgo className ={className} date={start}/>
```

Comment:
Instead of a new component, you should be able to return `styled(TimeAgo)` and add type parameters directly if typescript complains.

For example:
```
const NewHeader = styled(Header)<{ customColor: string }>`
  color: ${props => props.customColor};
`
```